### ISSUE TITLE
Onionshare 2.6.2 won't connect to tor

### ISSUE THREAD

**Issue by ForgottenBeast** [NixOS/nixpkgs#356591]
## Describe the bug

When onionshare tries to connect to a Tor daemon using the control socket, the progress gets stuck at 0% Connecting to the Tor network.

## Steps To Reproduce

To Reproduce

-    Set these settings:
-    Connect using socket file
-    Socket file = /var/run/tor/control
-    SOCKS port = 127.0.0.1, 9050
-   Try to connect (or test connection).

### Debug logs
Output of the GUI with --verbose

When onionshare initially tries to connect, the Tor daemon logs Aug 11 23:25:51.000 [notice] New control connection opened., indicating that it assumes a successful connection.

## Expected behavior

Onionshare connects to tor.

## Metadata

onionshare issue [1913](https://github.com/onionshare/onionshare/issues/1913) opened by @LordMZTE 

## Notify maintainers

@bbjubjub2494 

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by ForgottenBeast** [comment]
FWIW version 2.5 from c2c0373ae7abf25b7d69b2df05d3ef8014459ea3 works almost flawlessly (chat function is broken)

**Comment by ForgottenBeast** [comment]
after testing previous versions:
2.5 crashes after connecting to tor
2.4 works

**Comment by adam248** [comment]
I do believe this is a NixOS sided issue.

for anyone hit with this issue ->
2.6.3 flatpak works on NixOS

**Comment by Ate329** [comment]
> I do believe this is a NixOS sided issue.
> 
> for anyone hit with this issue -> 2.6.3 flatpak works on NixOS

Yes I tried it myself and I can confirm that the flatpak version works perfectly.
Here is the link if anyone is interested: [OnionShare](https://flathub.org/apps/org.onionshare.OnionShare)
